lib/kafka/consumer.rb

```diff
@@ -44,7 +49,8 @@ module Kafka
   #
   class Consumer
 
-    def initialize(cluster:, logger:, instrumenter:, group:, fetcher:, offset_manager:, session_timeout:, heartbeat:, refresh_topic_interval: 0)
+    def initialize(cluster:, logger:, instrumenter:, group:, fetcher:, offset_manager:,
+                   session_timeout:, heartbeat:, refresh_topic_interval: 0, interceptors: nil)
```

Comment:
Could it be possible to have a NoOp interceptor by default instead of nil?

For example, I would except `ConsumerInterceptors.new(interceptors: [], logger: logger)` to do nothing when called.

Then you don't need to do `unless interceptors.nil?` everywhere in the code.